### ISSUE TITLE
Selenium-webdriver js Bindings doesn't print all stack trace on error

### ISSUE THREAD

**Issue by niyarlatotep** [SeleniumHQ/selenium#7626]
## 🐛 Bug Report
Selenium-webdriver js bindings doesn't print all stack trace on error

## To Reproduce
```
(async ()=>{
    try {
        const driver = new Builder().forBrowser('chrome').build();
        await driver.get('https://www.google.com/');
        await driver.findElement(By.css('nonexistent'));
    } catch (e) {
        console.log(e.stack)
    }
})();
```
## Expected behavior

Error stack points at line ```await driver.findElement(By.css('nonexistent'));```

it worked well in version 3.x.x

## Environment

OS: Windows 10
Browser: Chrome
Browser version: 77.0.3865.90
Browser Driver version: ChromeDriver 77
Language Bindings version: Node js 4.0.0-alpha.5

**Comment by ProTip** [comment]
I have created my own workaround using a proxy object to record the stack when functions are called on the webdriver and creates a composite error if the target function throws and error. It works well and produces correct line and source output in Jest:

https://github.com/rundeck/rundeck/pull/6355/files

**Comment by alex028502** [comment]
Here is my workaround. https://gitlab.com/alex028502/binary-tools/-/blob/master/ip-address/fix-selenium-stack-trace/index.js
I just wrap every single call with this function I created that stores the current stack trace info, and then appends it to any errors that get raised. (you also need at least node 12 I think)

**Comment by potapovDim** [comment]
Hey, @niyarlatotep, is it still relevant ?

**Comment by niyarlatotep** [comment]
@potapovDim don't know, haven't used since then.

**Comment by alex028502** [comment]
```
$ mkdir new-selenium-stacktrace-experiment
$ cd new-selenium-stacktrace-experiment
$ npm install selenium-webdriver
$ cat node_modules/selenium-webdriver/package.json | grep version
  "version": "4.0.0",
$ cat > index.js 

const webdriver = require('selenium-webdriver');
const Builder = webdriver.Builder;
const By = webdriver.By;

(async ()=>{
    try {
        const driver = new Builder().forBrowser('chrome').build();
        await driver.get('https://www.google.com/');
        await driver.findElement(By.css('nonexistent'));
    } catch (e) {
        console.log(e.stack)
    }
})();

$ node index.js 
NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":"nonexistent"}
  (Session info: chrome=95.0.4638.54)
    at Object.throwDecodedError (~/new-selenium-stacktrace-experiment/node_modules/selenium-webdriver/lib/error.js:517:15)
    at parseHttpResponse (~/new-selenium-stacktrace-experiment/node_modules/selenium-webdriver/lib/http.js:643:13)
    at Executor.execute (~/new-selenium-stacktrace-experiment/node_modules/selenium-webdriver/lib/http.js:569:28)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async thenableWebDriverProxy.execute (~/new-selenium-stacktrace-experiment/node_modules/selenium-webdriver/lib/webdriver.js:732:17)
    at async ~/new-selenium-stacktrace-experiment/index.js:10:9
```

It seems to be fixed! (for this example at least)

Thanks a lot

**Comment by alex028502** [comment]
Even when I go back to `4.0.0-alpha.5`, it still seems to work.  (with chrome 95).
However, I saw the same thing, which is how I found this thread.

So I guess that either:
a) there are other examples which still cause this problem
b) I was using return instead of await by accident (if you put return instead of the last await, the stack trace doesn't have the info, but that is expected)
c) other dependencies like the chrome version somehow caused the issue
